packages/config/src/handlers/application/applicationConfigHandler.ts

```diff
@@ -27,29 +27,24 @@ export default async function configHandler(
   request: Hapi.Request,
   h: Hapi.ResponseToolkit
 ) {
-  try {
-    const [certificates, config, systems] = await Promise.all([
-      getActiveCertificatesHandler(request, h).then((certs) =>
-        Promise.all(
-          certs.map(async (cert) => ({
-            ...cert,
-            svgCode: await getDocumentUrl(cert.svgCode, {
-              Authorization: request.headers.authorization
-            })
-          }))
-        )
-      ),
-      getApplicationConfig(request, h),
-      getSystems(request, h)
-    ])
-    return {
-      config,
-      certificates,
-      systems
-    }
-  } catch (ex) {
-    logger.error(ex)
-    return {}
+  const [certificates, config, systems] = await Promise.all([
+    getActiveCertificatesHandler(request, h).then((certs) =>
+      Promise.all(
+        certs.map(async (cert) => ({
+          ...cert,
+          svgCode: await getDocumentUrl(cert.svgCode, {
+            Authorization: request.headers.authorization
+          })
+        }))
+      )
+    ),
+    getApplicationConfig(request, h),
+    getSystems(request, h)
+  ])
+  return {
+    config,
+    certificates,
+    systems
```

Comment:
@kayumuzzaman Let's keep it how it was before as I'm pretty sure it was like that for a reason that we just can't remember now. Instead add something like this:
```
if (process.env.NODE_ENV === 'development') {
  throw ex
}
```